erpnext/manufacturing/doctype/bom/bom.js

```diff
@@ -126,6 +126,12 @@ frappe.ui.form.on("BOM", {
 		}
 	},
 
+	item: function(frm) {
+		frappe.db.get_value('Item', {name: frm.doc.item}, 'inspection_required_before_manufacturing', (r) => {
+			frm.set_value("inspection_required", r.inspection_required_before_manufacturing)
```

Comment:
Also pull in the QI template if it's set in the Item

Reply:
It will automatically pull once this checkbox is set

Reply:
Try creating a new BOM from the search bar, not through the item